gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/WorkInProgress.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gobblin.service.modules.orchestration;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Custom Annotation for classes that are under development.
+ * It will make the classes available only during compilation phase, and not in the build.
+ */
```

Comment:
doesn't guava have an `@Alpha` annotation to convey similar?

I'm not fully clear there's a need to prevent classes w/ this annotation from going into the build.  please justify if you perceive a risk we should work to avoid.

Reply:
I have used `@Alpha` in the classes and deleted `@WorkInProgress`